### ISSUE TITLE
Go Client Example?

### ISSUE THREAD

**Issue by kpfaulkner** [lonng/nano#56]
Hi

I'm just wondering if there is a Go client example (not HTML+JS) anywhere?

Thanks

Ken

**Comment by lonng** [comment]
@kpfaulkner There is no go client example, sorry for that.

**Comment by kpfaulkner** [comment]
Am I right in saying that if I find a go client that is pomelo  compatible, then in theory it would work?

**Comment by lonng** [comment]
@kpfaulkner Yes, and the examples contain a tough implementation, maybe you can refine it and make it ready for production.
